iotdb-core/datanode/src/main/java/org/apache/iotdb/db/pipe/connector/protocol/pipeconsensus/PipeConsensusAsyncConnector.java

```diff
@@ -521,4 +551,25 @@ public int getTransferBufferSize() {
   public int getRetryBufferSize() {
     return retryEventQueue.size();
   }
+
+  @Override
+  public long getLocalUserWriteProgress() {
+    PipeStaticMeta staticMeta = PipeAgent.task().getPipeStaticMeta(consensusPipeName);
+    String committerKey =
+        String.format(
+            "%s_%s_%s", staticMeta.getPipeName(), consensusGroupId, staticMeta.getCreationTime());
+    return PipeEventCommitManager.getInstance().getGivenConsensusPipeCommitId(committerKey);
+  }
+
+  @Override
+  public long getCurrentPipeReplicateProgress() {
```

Comment:
What's the difference between it and the "writeProgress"?

Reply:
One is the synchronization progress of the pipe event, and the other is the progress of the user writing the event